packages/pluggableWidgets/gallery-web/src/components/__tests__/Gallery.spec.tsx

```diff
@@ -0,0 +1,131 @@
+import { createElement } from "react";
+import { mount, shallow } from "enzyme";
+import { Gallery, GalleryProps } from "../Gallery";
+
+const itemWrapperFunction = ({
+    onClick = false,
+    customClass,
+    customOnClick
+}: {
+    onClick?: boolean;
+    customClass?: string;
+    customOnClick?: () => void;
+}): GalleryProps<string>["itemRenderer"] => (wrapper, item) =>
+    wrapper(item, customClass, onClick ? (customOnClick ? customOnClick : () => console.log("Clicked")) : undefined);
```

Comment:
Why `customOnClick` and boolean `onClick`? Can't you just use `customOnClick` to check? If you don't need it, you can overwrite with `undefined`